index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- the wraping container -->
+    <div class="parent-container">
```

Comment:
- [ ] Let's fix the spacing between element so that it matches your chosen template.

Your page:
<img width="221" alt="Loom_4dhgBI5b7U" src="https://user-images.githubusercontent.com/81468675/149381506-fcee7d1d-d833-47ab-9fce-14e4b33bea6d.png">

Figma's:
<img width="187" alt="Loom_FzzNrdmmql" src="https://user-images.githubusercontent.com/81468675/149381557-c5dd04df-2225-4b03-8320-cf51a03c0a5d.png">